crates/re_string_interner/src/lib.rs

```diff
@@ -169,7 +169,8 @@ impl StringInterner {
         // and that is wat `ier()` gives us, so this is all correct.
         self.map
             .iter()
-            .map(|(k, v)| std::mem::size_of_val(k) + std::mem::size_of_val(v) + v.len())
+            // NOTE: v is a `&&str`!
+            .map(|(k, v)| std::mem::size_of_val(k) + std::mem::size_of_val(*v) + v.len())
```

Comment:
```suggestion
            .map(|(k, v: &&str)| std::mem::size_of_val(k) + std::mem::size_of_val(v) + v.len())
```

we want to measure the size of `&str`, NOT the size of `str`. `size_of_val` takes a _reference_ to what it is measuring the size of, so `size_of_val(v)` is actually correct.

We could just go with `std::mem::size_of::<&str>()` to avoid confusion though:

```suggestion
            .map(|(k, _: &&str)| std::mem::size_of_val(k) + std::mem::size_of::<&str>() + v.len())
```